filetype/types/archive.py

```diff
@@ -655,7 +657,8 @@ def __init__(self):
 
     @staticmethod
     def _to_little_endian_int(buf):
-        return int(bytearray(reversed(buf)).hex(), 16)
+        # return int.from_bytes(buf, byteorder='little')
```

Comment:
Heh, I didn't know this one — nice.

Reply:
But I am not sure if this works in py2.x or not :(